### DISCUSSION TITLE
Sophisticated Barrels, please?

### DISCUSSION THREAD

**Discussion by Wakatsu** [P3pp3rF1y/SophisticatedBackpacks#280] [Feature / Idea Archive] [answered]
So, might be a dumb idea, but what if all these functions, but in a form of a full block non-wearable barrel?

Here are my reasons for this:
As of now, Sophisticated Backpacks are the single best mobile storage mod ever existed in history of mobile storage mods for minecraft, it is balanced, intuitive, customizable, it's everything one could ask for. Meanwhile, every builder and survivalist knows well the pain of organizing the storage systems back at your base, to store the spoils of your victories. These backpacks solve 97 out of 100 problems on that topic, the only downside being - a wall full of backpacks looks hella weird.

So, how about Sophisticated Barrels for all of us base-building enthusiasts? It should be easy to achieve, since it should pretty much be a model swap, and minus the ability to equip the barrel on character (altho, character carrying a barrel on their back would look dope, so maybe consider that too).

I know I would kill to have such barrels for my base storage solution.

**Comment by Seikikai** [comment]
![imagen](https://user-images.githubusercontent.com/54079357/139744020-666ab845-96fd-4f5d-a7bd-2cdd910c5f43.png)
I Agree, This Is One Of The Best Storage Management Mods I Ever Seen

**Comment by P3pp3rF1y** [comment, marked as answer]
there is now sophisticated storage so I would consider this done :D
